lms/djangoapps/mobile_api/course_info/tests.py

```diff
@@ -9,35 +9,33 @@
 
 from courseware.tests.factories import UserFactory
 from xmodule.html_module import CourseInfoModule
+from xmodule.modulestore import ModuleStoreEnum
 from xmodule.modulestore.django import modulestore
 from xmodule.modulestore.tests.factories import CourseFactory
-from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.xml_importer import import_from_xml
 
+from ..testutils import (
+    MobileAPITestCase, MobileCourseAccessTestMixin, MobileEnrolledCourseAccessTestMixin, MobileAuthTestMixin
+)
 
```

Comment:
More unused imports. Seems like pylint is not picking these up.

`import json
from django.core.urlresolvers import reverse
from rest_framework.test import APITestCase

from courseware.tests.factories import UserFactory

from xmodule.modulestore.tests.factories import CourseFactory`